scripts/netContain/ReleaseContainer.sh

```diff
@@ -0,0 +1,15 @@
+if [ "$1" = "" ]; then
+  echo "Please provide the contiv release version to use"
+  echo "Usage: ./ReleaseContainer.sh <contiv version>"
+  echo "Example: ./ReleaseContainer.sh v0.1-11-30-2016.20-08-20.UTC"
+  echo "Released versions are available from https://github.com/contiv/netplugin/releases"
+  exit 1
+else
+  contivVersion=$1
+fi
+echo "Login to docker hub as contiv user"
+docker login -u contiv
```

Comment:
Thanks for taking care of this script. Appreciate it !
one small comment, instead of fixing username as contiv, can we have default username as contiv but if you want we can use your own username and password for dockerhub ?